data/avro-schema/src/main/resources/avro/net/corda/data/membership/rpc/response/MGMGroupPolicyResponse.avsc

```diff
@@ -0,0 +1,48 @@
+{
+  "type": "record",
+  "name": "MGMGroupPolicyResponse",
+  "namespace": "net.corda.data.membership.rpc.response",
+  "doc": "MGM Group Policy Response",
+  "fields": [
+    {
```

Comment:
This should just return a string. The string will be the full group policy. No need to separate it out in to it's different components here. If the group policy changes we would have to change this schema also.

Reply:
e.g.
```
{
  "type": "record",
  "name": "MGMGroupPolicyResponse",
  "namespace": "net.corda.data.membership.rpc.response",
  "doc": "MGM Group Policy Response",
  "fields": [
    {
      "name": "groupPolicy",
      "doc": "The full group policy JSON string.",
      "type": "string"
    }
  ]
}
```